crates/bevy_core_pipeline/src/skybox/skybox_prepass.wgsl

```diff
@@ -0,0 +1,45 @@
+#import bevy_render::view::View
+#import bevy_core_pipeline::fullscreen_vertex_shader::FullscreenVertexOutput
+
+struct PreviousViewUniforms {
+    inverse_view: mat4x4<f32>,
+    view_proj: mat4x4<f32>,
+}
+
+@group(0) @binding(0) var<uniform> view: View;
+@group(0) @binding(1) var<uniform> previous_view: PreviousViewUniforms;
+
+// This vertex shader produces the following, when drawn using indices 0..3:
+//
+//  1 |  0-----x.....2
+//  0 |  |  s  |  . ´
+// -1 |  x_____x´
+// -2 |  :  .´
+// -3 |  1´
+//    +---------------
+//      -1  0  1  2  3
+//
+// The axes are clip-space x and y. The region marked s is the visible region.
+// The digits in the corners of the right-angled triangle are the vertex
+// indices.
+//
+// The top-left has UV 0,0, the bottom-left has 0,2, and the top-right has 2,0.
+// This means that the UV gets interpolated to 1,1 at the bottom-right corner
+// of the clip-space rectangle that is at 1,-1 in clip space.
+@vertex
+fn vertex_main(@builtin(vertex_index) vertex_index: u32) -> FullscreenVertexOutput {
```

Comment:
Is there a way to reuse this? https://github.com/bevyengine/bevy/blob/main/crates/bevy_core_pipeline/src/fullscreen_vertex_shader/fullscreen.wgsl

Reply:
Perhaps like this pass: https://github.com/bevyengine/bevy/blob/555963297719a5125d6603905351fe99f0b1f78a/crates/bevy_core_pipeline/src/fxaa/mod.rs#L179